include/zephyr/bluetooth/gatt.h

```diff
@@ -72,6 +72,32 @@ enum {
 	 *  passed to write callback.
 	 */
 	BT_GATT_PERM_PREPARE_WRITE = BIT(6),
+
+	/** @brief Attribute write permission with 128bit entropy key connection.
+	 *
+	 *  If set, write access requires connection using an at least 128bit entropy
+	 *  key derived from one of the following sources:
+	 *  - LE Secure Connections
+	 *  - OOB method
+	 *
+	 *  This permission is a superset of encryption, meaning if set
+	 *  BT_GATT_PERM_READ_ENCRYPT is implied, and can be omitted. It does
+	 *  however not imply BT_GATT_PERM_READ_AUTHEN.
+	 */
+	BT_GATT_PERM_READ_128B_ENTROPY = BIT(7),
+
+	/** @brief Attribute write permission with 128bit entropy key connection.
+	 *
+	 *  If set, write access requires connection using an at least 128bit entropy
+	 *  key derived from one of the following sources:
+	 *  - LE Secure Connections
+	 *  - OOB method
+	 *
+	 *  This permission is a superset of encryption, meaning if set
+	 *  BT_GATT_PERM_WRITE_ENCRYPT is implied, and can be omitted. It does
+	 *  however not imply BT_GATT_PERM_WRITE_AUTHEN.
```

Comment:
Similar text should be applied to `BT_GATT_PERM_WRITE_AUTHEN`

Reply:
Its been added